packages/react-solid-state/src/index.js

```diff
@@ -23,7 +24,7 @@ let inSolidEffect = false;
 function trackNesting(args) {
   const fn = args[0];
   return [
-    function() {
+    function () {

```

Comment:
just realized this got added from prettier in my IDE - can revert if not desired

Reply:
Don't worry about it.